webapp/app.py

```diff
@@ -1,33 +1,47 @@
+import os
+
 from apispec import APISpec
 from apispec.ext.marshmallow import MarshmallowPlugin
 from canonicalwebteam.flask_base.app import FlaskBase
-from flask import jsonify
 from flask_apispec.extension import FlaskApiSpec
+from flask_sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
+
+
+def create_app():
```

Comment:
Could you try removing this wrapping function? I'm not sure it's necessary in this case, and I'm [not a big fan](https://github.com/canonical-web-and-design/snapcraft.io/issues/1850) of the pattern in general.

Reply:
No longer using flask-testing. The create_app method is no longer.